src/library-authoring/add-content/AddContentContainer.tsx

```diff
@@ -179,10 +188,8 @@ const AddContentContainer = () => {
       blockType,
       definitionId: `${uuid4()}`,
     }).then((data) => {
+      linkComponent(data.id);
```

Comment:
Thank you for fixing the "create block in collection" issue here too @navinkarkera ! 🚀